ltr/sagemaker/deploy.py

```diff
@@ -0,0 +1,115 @@
+import sagemaker
+import sagemaker.tensorflow
+import sys
+import time
+
+def deploy(model_name, s3_bucket, role, govuk_environment, instance_count, instance_type):
+    endpoint_name = f"govuk-{govuk_environment}-search-ltr-endpoint"
+
+    session = sagemaker.Session()
+
+    update_endpoint = False
+
+    # try to find the current endpoint config
+    try:
+        current_endpoint_config_name = session.boto_session.client(
+            "sagemaker"
+        ).describe_endpoint(EndpointName=endpoint_name)["EndpointConfigName"]
+
+        current_endpoint_config = session.boto_session.client(
+            "sagemaker"
+        ).describe_endpoint_config(EndpointConfigName=current_endpoint_config_name,)
```

Comment:
nit: stray comma makes this look a bit odd (like a 1-tuple)
```suggestion
        ).describe_endpoint_config(EndpointConfigName=current_endpoint_config_name)
```